src/stable/build/utils/dfx.ts

```diff
@@ -2,9 +2,11 @@ import { HttpAgent } from '@dfinity/agent';
 import { Secp256k1KeyIdentity } from '@dfinity/identity-secp256k1';
 import { execSync } from 'child_process';
 
+import { getDfxJsonDirPath } from '#utils/global_paths';
+
 export function getCanisterId(canisterName: string): string {
     return execSync(
```

Comment:
Use the parameters of execSync to set the working directory instead of doing a `cd`

Reply:
Done